ambry-replication/src/main/java/com/github/ambry/replication/ReplicationMetrics.java

```diff
@@ -171,7 +171,12 @@ public class ReplicationMetrics {
   // This metric tracks cross colo get requests bytes rate for Standby replicas.
   public final Map<String, Meter> interColoReplicationFetchBytesRateForStandbyReplicas = new ConcurrentHashMap<>();
 
+  public static final String BACKUP_INTEGRITY_ERROR = "BackupIntegrityError";
+  public final Counter backupIntegrityError;
+
   public ReplicationMetrics(MetricRegistry registry, List<? extends ReplicaId> replicaIds) {
+    backupIntegrityError =
+        registry.counter(MetricRegistry.name(ReplicationMetrics.class, BACKUP_INTEGRITY_ERROR));
```

Comment:
we usually just put the metric name in the register.meter.

Reply:
sorry i don't get it, do you mean to change the class name of the metric ?

Reply:
I mean you don't need to create a static final string variable for the metric name.